openassessment/templates/openassessmentblock/edit/oa_edit.html

```diff
@@ -157,9 +157,9 @@ <h2 class="openassessment_alert_title">{% trans "Rubric Change Impacts Settings
 
             <p class="openassessment_description" id="openassessment_step_select_description">
                 {% if 'example_based_assessment' in editor_assessments_order %}
-                    {% trans "In this assignment, you can include steps for learner training, peer assessment, self assessment, and example based assessment. Select the steps that you want below, and then drag them into the order that you want. If you include an example based assessment step, it must precede all other steps. If you include a learner training step, it must precede peer and self assessment steps." %}
+                    {% trans "In this assignment, you can include learner training, peer assessment, self assessment, example based assessment, and staff assessment steps. Select the steps that you want below, and then drag them into the order you want. If you include an example based assessment step, it must precede all other steps. If you include a learner training step, it must precede peer and self assessment steps. If you include a staff assessment step, it must be the final step. The type and number of assessment steps cannot be changed after a problem has been released." %}
                 {% else %}
-                    {% trans "In this assignment, you can include steps for learner training, peer assessment, and self assessment. Select the steps that you want below, and then drag them into the order that you want. If you include a learner training step, it must precede all other steps." %}
+                    {% trans "In this assignment, you can include learner training, peer assessment, self assessment, and staff assessment steps. Select the steps that you want below, and then drag them into the order you want. If you include a learner training step, it must precede all other steps. If you include a staff assessment step, it must be the final step. The type and number of assessment steps cannot be changed after a problem has been released." %}
```

Comment:
@catong Just a heads up - I had to modify this string twice, with a slight difference depending on whether example-based is included or not.